.github/workflows/push.yml

```diff
@@ -20,6 +20,11 @@ jobs:
       - name: Maven Build
         run: mvn clean spring-javaformat:validate package
 
+      - name: Check generated files are up to date
```

Comment:
Legit questions!

> Can be that the API is changed and the open API is not updated?
no, because (see next answer :) )
> Are we creating the open API when we build?
yes!
> Are we updating the auto-generation of the SDK on package?
yes

To recap: at the moment during every `package` phase, the generator first creates the `openapi.json` file from the spring beans, then it generates the SDK source code and the jar. 

In future PR, I'll move the generation to a following phase (maybe `verify`) to let developer work on their changes without having to deal with all the generated files. Then, the CI will verify if the outcome of `mvn verify` has uncommited changes.